src/data/faq.json

```diff
@@ -2869,9 +2873,9 @@
                     "description": "Since version 2.2 of the Corona-Warn-App you are able to create a rapid test profile with your data such as first and last name, date of birth, address, phone number and e-mail address. Using a rapid test profile, you ensure that registration at a testing center is faster. The profile contains a QR code, which you can show at the testing center. However, you should still have your ID card ready."
                 },
                 {
-                    "term": "Recovery Certificate",
-                    "anchor": "recovery_certificate",
-                    "description": "The certificate of recovery is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                    "term": "Recovery certificate",
+                    "anchor": "recovery_certificate_alt",
+                    "description": "Please see the gelossary entry <a href='#recovery_certificate'>Certificate of Recovery</a>."

```

Comment:
"gelossary" is misspelled

Reply:
Also this is a dead link. It should be #glossary_recovery_certificate

Reply:
Fixed with [`507e44b`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/507e44b7918968071818c73a68239dcbecd904dd).